docs/se/introduction.adoc

```diff
@@ -157,6 +157,30 @@ Enables Java applications to participate in WebSocket interactions as both serve
 ====
 
 
+
+== Migration
+
+In case you need to upgrade the version of Helidon, follow the `upgrade guides`.
```

Comment:
these are migration guides not upgrade guides right?

Reply:
I see both and I'm not sure if that's intentional. What is the difference?

Reply:
keeping it `migration`